kmir/src/tests/integration/data/prove-rs/show/symbolic-args-fail.expected

```diff
@@ -27,13 +55,38 @@
    ├─ 5
    │   #selectBlock ( switchTargets ( ... branches: branch ( 0 , basicBlockIdx ( 3 ) )
    │
-   │  (66 steps)
-   └─ 7 (stuck, leaf)
-       #applyUnOp ( unOpPtrMetadata , typedValue ( Reference ( 0 , place ( ... local: l
-       span: 59
+   │  (87 steps)
+   ├─ 7
+   │   #selectBlock ( switchTargets ( ... branches: branch ( 0 , basicBlockIdx ( 7 ) )
+   ┃
+   ┃ (1 step)
+   ┣━━┓
+   ┃  │
+   ┃  ├─ 11
+   ┃  │   #execBlockIdx ( basicBlockIdx ( 7 ) ) ~> .K
+   ┃  │
+   ┃  │  (4 steps)
+   ┃  ├─ 16 (terminal)
+   ┃  │   #EndProgram ~> .K
+   ┃  │
+   ┃  ┊  constraint: true
+   ┃  ┊  subst: ...
+   ┃  └─ 2 (leaf, target, terminal)
+   ┃      #EndProgram ~> .K
+   ┃
+   ┣━━┓
+   ┃  │
+   ┃  ├─ 12
+   ┃  │   #selectBlock ( switchTargets ( ... branches: .Branches , otherwise: basicBlockId
+   ┃  │
+   ┃  │  (85 steps)
+   ┃  └─ 17 (vacuous, leaf)
+   ┃      #expect ( typedValue ( BoolVal ( 0 <Int size ( ARG_ARRAY7:List ) ) , ty ( 28 ) ,
+   ┃
+   ┗━━┓
+      │
+      └─ 13 (stuck, leaf)
+          #selectBlock ( switchTargets ( ... branches: branch ( 0 , basicBlockIdx ( 7 ) )
```

Comment:
We can see we're getting `vacuous` and `stuck` nodes (which are nonsensical), and that our `#switchBlocks` is not carrying through side-conditions correctly. That will have to be investigated in a further PR.